source/sass/component/_tooltip.scss

```diff
@@ -1,125 +1,140 @@
 .c-tooltip {
```

Comment:
Define local css variables for colors. default to alpha color and contrasting color.